certbot/tests/util.py

```diff
@@ -276,6 +276,15 @@ def _write_msg(message, *unused_args, **unused_kwargs):
         """
         stdout.write(message)
 
+    def mock_method(*args, **kwargs):
+        """
+        Mock function for IDisplay methods.
+        """
+        message = args[0] if args else kwargs.get('message', None)
+        if message:
+            _write_msg(message, args, kwargs)
+        _assert_valid_call(args, kwargs)
```

Comment:
Unless there's another reason to do it this way that I'm missing, I think we should call `_assert_valid_call` first as this is the behavior of the methods in `certbot/display/util.py`.